### DISCUSSION TITLE
请问手机版本怎么设置落地代理才能生效？有没有大佬告知一下。

### DISCUSSION THREAD

**Discussion by ya950** [2dust/v2rayNG#3949] [General]
😭

**Comment by 2dust** [comment]
https://github.com/2dust/v2rayN/wiki/%E5%89%8D%E7%BD%AE%E4%BB%A3%E7%90%86%E5%92%8C%E8%90%BD%E5%9C%B0%E4%BB%A3%E7%90%86%E8%AF%B4%E6%98%8E

**Reply by ya950** [reply]
试了，手机没有作用，IP还不固定，电脑节点开singbox  可以，节点xray类型也是不固定的，![Screenshot_20241115_202004.jpg](https://github.com/user-attachments/assets/f20b0aae-c141-408b-be28-b43c78a9d25e)

![Screenshot_20241115_202119.jpg](https://github.com/user-attachments/assets/43f6a682-1021-419a-8197-e11e48cc2eb1)

![Screenshot_20241115_202155.jpg](https://github.com/user-attachments/assets/3aea7d84-c1d6-440e-a8fd-c3fb18b4ea09)
